index.js

```diff
@@ -5,16 +5,20 @@ const express = require('express'),
       displayRoutes = require('express-routemap'),
       session = require('express-session'),
       bcrypt = require('bcrypt'),
-      morgan = require('morgan');
+      morgan = require('morgan'),
+      Sequelize = require('sequelize');
 
 var app = express(),
-    db = require('./models');
+    db = require('./models'),
+    sequelize = new Sequelize('moviematch', process.env.DB_USERNAME, process.env.DB_PASSWORD, { dialect: 'postgres' });
+
+var userRouter = require('./routes/user'),
+    adminRouter = require('./routes/admin');
 
-var userRouter = require('./routes/user');
 
 app.use(express.static('public'));
 app.use(morgan('dev'));
-app.use(bodyParser.urlencoded({ extended: false}));
+app.use(bodyParser.urlencoded());
```

Comment:
add extendend:true, this way you can support nested forms like input(type="text" name="user[email]")